saleor/graphql/discount/mutations/voucher/voucher_update.py

```diff
@@ -35,7 +36,11 @@ class Meta:
             WebhookEventInfo(
                 type=WebhookEventAsyncType.VOUCHER_UPDATED,
                 description="A voucher was updated.",
-            )
+            ),
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.VOUCHER_UPDATED,
```

Comment:
`type=WebhookEventAsyncType.VOUCHER_UPDATED,`  is it correct ?  (same as above one)🤔